weaviate/rbac/models.py

```diff
@@ -488,6 +533,24 @@ def delete(*, collection: Optional[str] = None) -> _CollectionsPermission:
         )
 
 
+class _TenantsFactory:
+    @staticmethod
+    def create(*, collection: Optional[str] = None) -> TenantsPermission:
+        return TenantsPermission(collection=collection or "*", action=TenantsAction.CREATE)
```

Comment:
Will providing a tenant here be featured in a further PR? `PermissionsTenants` has the `tenant` attribute but I see that `TenantsPermission` doesn't

Reply:
yes, didn't we remove tenant filtering from all factory methods until we have properly evaluated it? :)

Reply:
That's good then, let's merge!